guides/GuideForGuests.md

```diff
@@ -0,0 +1,88 @@
+
+#### Questions on the call
+
+During the call, your host will ask some combinations of the following questions:
+
+* Your name
+* Name of the project
+* Project website(s)
+* What problem is your project trying to solve?
+* How did you get started with the project?
+* What is fun about the project?
+* How can someone get involved? What do you need help with?
+
+We'd like to also highlight ***how openness has enabled or strengthened the project***. Please think about how your answers to the questions above can cover this goal (without needing an explicit "what in your project is open" question).
+
+If there are any questions you'd particularly like the host to ask, please let her know in advance.
+
+#### Details for the OHBM blog
+
+After the call we will archive the YouTube video on the [OHBM blog](https://www.ohbmbrainmappingblog.com/). In advance of the call please complete [this google form](https://goo.gl/forms/3foIuNP9G7kmCW193) with the following information:
+
+* Your name
+* Name of the project
+* Project website(s)
+* 1 sentence about the project
+* 1 sentence about how someone can get involved (including contact information)
+
+The structure of the blog will be three introductory sentences, the video, and then bullet points with the information above for the three guest projects. We'd like to turn this around very quickly, so please try to provide the sentences in advance rather than after the call. Here's the google form link again: https://goo.gl/forms/3foIuNP9G7kmCW193
+
+### During the call
+
+***Please join the call 5 minutes before the offical start time so we're all ready to go!***
+
+The call will start with a brief welcome, and then we'll jump into the conversations. These should last between 8 and 10 minutes. Your host will contact you in advance to discuss the running order.
+
+#### Sharing your screen
+
+It's very easy to share your screen from within Google Hangouts, and this might be a really great way to orient our viewers to your project. You could navigate to the project webpage, or showcase an example.
+
+You do not need to prepare a presentation for this call. In fact, we'd prefer that you didn't! It will be much more fun to see your face when you're describing the project. We're confident that you know an awful lot about it and don't need the slides. However, if there are some images that would particularly help to describe your project goals to our audience, then feel free to have them ready to share on your screen.
+
+#### Additional guests
+
+It is very likely that your project is a collaborative endeavor. If you'd like to invite other representatives of the project to the call that's absolutely fine, just remember that we don't have a huge amount of time, so probably more than three people per project will be difficult to wrangle efficiently.
+
+The additional guests can either be in the same place as you (and therefore connect on the same screen) or they can join the google hangout individually.
```

Comment:
If this is true then we should revise the previous sentence that only 4 people will be on the call. I think that could get a little confusing for viewers unless explicitly announced, though.

Reply:
I'm not sure I follow. I don't think anything will be confusing for the viewers because the host will introduce everyone at the start. The main point to emphasise is that the viewers who are watching live will *not* be on the call (which I think will seem oddly empty at first!).

Reply:
Ah, ok ! I think I misunderstood.